srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.xml

```diff
@@ -797,15 +797,26 @@ SRTP packets                 <------->      SRTP packets
       <t>The EKT Cipher includes its own authentication/integrity
       check. For an attacker to successfully forge a full EKT packet,
       it would need to defeat the authentication mechanisms of the EKT
-      Cipher authentication mechanism.</t>
+      Cipher authentication mechanism. </t>
 
       <t>The presence of the SSRC in the EKTPlaintext ensures that an
       attacker cannot substitute an EKTCiphertext from one SRTP stream
       into another SRTP stream.</t>
 
-      <t>An attacker who tampers with the bits in FullEKTField can
-      prevent the intended receiver of that packet from being able to
-      decrypt it. This is a minor denial of service vulnerability.</t>
+     <t>An attacker who tampers with the bits in FullEKTField can
+     prevent the intended receiver of that packet from being able to
+     decrypt it. This is a minor denial of service
+     vulnerability. Similarly the attacker could take an old
+     FullEKTField from the same session and attach it to the
+     packet. The FullEKTField would correctly decode and pass
+     integrity but the key extracted from the FullEKTField , when used
+     to decrypt the SRTP payload, would be wrong and the SRTP
+     integrity check would fail. Note that the FullEKTField only
+     changes the decryption key with and does not change the
+     encryption key. None of these are considered significant attacks
+     as any attacker that can modify the packets in transit, and cause
+     the integrity check to fail or the attacker could just drop the
```

Comment:
The "or" starts to make the sentence worry. Perhaps a period and new sentence.  Or remove it, because that's rather obvious.